playbooks/openstack/ansible.cfg

```diff
@@ -0,0 +1,24 @@
+# config file for ansible -- http://ansible.com/
```

Comment:
Is there anything in this ansible.cfg that is required that is not in /ansible.cfg ?

Reply:
I just had a look and not really. We've got some defaults we would lose, but that wouldn't be the end of the world.

There's one entry (inventory_ignore_extensions) I'd like to keep, but I'll just add that to the root ansible.cfg instead.